packages/publikator/graphql/schema-types.js

```diff
@@ -69,13 +69,20 @@ type RepoMeta {
   briefingUrl: String
 }
 
+input RepoDateRange {
+  from: DateTime!
+  before: DateTime!
```

Comment:
do you have very strong opinions against renaming `before` to `until` to `to`? Before rubs me the wrong way – this is just not how people speak and feels like I'm summoning **everything** with a `publishingDate < before`.

Reply:
Nope, no string objections.